lib/system/web/src/lib.rs

```diff
@@ -184,8 +185,8 @@ macro_rules! group {
 // ===================
 
 pub fn dyn_into<T,U>(obj :T) -> Result<U>
-where T : wasm_bindgen::JsCast + Debug,
-      U : wasm_bindgen::JsCast
+    where T : wasm_bindgen::JsCast + Debug,
+          U : wasm_bindgen::JsCast
```

Comment:
We usually don't make indent here, do we?. And the { should be at end of line

Reply:
It was AI!

Reply:
(Accidental Indent) :P